library/src/main/java/com/novoda/downloadmanager/lib/Request.java

```diff
@@ -246,6 +247,15 @@ public Request allowTarUpdates() {
     }
 
     /**
+     * When a ETag header is present, the application should check the integrity of the
+     * downloaded file, otherwise the current download won't be able to be resumed
```

Comment:
Correct, when resuming a download through eTags, the application has to check the integrity of the downloaded file as it could be possible that the server changes the file keeping the same eTag